AzureVMNoiseDataset2024.md

```diff
@@ -53,10 +67,39 @@ You can download the dataset here:  [**vm-noise-data**](https://github.com/Azure
 |5101.0|33.22|2024-05-20 01:44:24.070|2|
 
 #### Description
+
 This benchmarking data was collected from `2023-05-28` to `2024-09-23` for a set of VMs and organized using the hive partitioning layout.
-There are a series of `92` metrics collected from a series of `40` benchmarks. These metrics were collected from VMs in 3 dimensions: VM lifespan, VM type, and VM region.
-For VM lifespan, we categorized VMs into two classes, long and short. Long running VMs ran for the entire duration of the study. Short running VMs were only ran each benchmark one time before being reallocated.
-For VM types, we chose `D8s_v5` VMs and `B8ms` VMs.
-For VM regions, we chose `westus2` and `eastus`.
-There were three VMs allocated for each combination of dimensions.
-There are some periods of missing data caused by crashes on our management nodes.
\ No newline at end of file
+
+There are a series of `92` metrics collected from a series of `40` benchmarks.
+
+These metrics were collected from VMs in 3 dimensions:
+
+1. VM lifespan
+
+    For VM lifespan, we categorized VMs into two classes: `short` and `long`.
+
+    Long running VMs were provisioned once and ran for the entire duration of the study.
+
+    Short running VMs were only ran each benchmark one time before being reallocated.
+
+    The purpose of this dimension is to influence which backend host the VM was assigned to in order to increase samples across different backend hosts.
```

Comment:
Maybe also add a note about backend host information being omitted, but a general sense of how often we noted VM migrations taking place?  Or how many different backend hosts we sampled in total?